src/ui/views/calendar-view.ts

```diff
@@ -67,6 +67,15 @@ export class DataviewCalendarRenderer extends DataviewRefreshableRenderer {
 
         const sources: ICalendarSource[] = [querySource];
         const renderer = this;
+
+        const debounce = (func: (...args: any[]) => void, wait: number) => {
```

Comment:
Obsidian already comes with a global `debounce` function with this same signature.